creator-node/src/utils/fixInconsistentUsers.ts

```diff
@@ -0,0 +1,24 @@
+import * as models from '../models'
+
+export const fixInconsistentUser = async (userId: string) => {
```

Comment:
also lets change userId to userUUID? userId could be overloaded cause there is a proper id that's an int